.travis.yml

```diff
@@ -15,5 +15,10 @@ install:
     - docker-compose up -d postgres elasticsearch redis selenium-server php-fpm webserver
     - docker-compose exec php-fpm composer install --no-interaction
 
-script:
-    - docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev tests-acceptance
+jobs:
+    include:
+        -   stage: Tests
+            name: Build all
+            script: docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev tests-acceptance
+        -   name: Build tests only
+            script: docker-compose exec php-fpm ./phing test-db-create tests tests-acceptance
```

Comment:
Why do you run tests twice? (part of build-demo-dev build and separately)

Reply:
Because I want to test if tests are working without "production" database. But we could build application in first stage and tests in second stage. But this is on you guys.

Reply:
I understand and I agree with you – this new feature should be tested to ensure this will work in the future.

Do you need to somehow prepare the application before `test-db-create tests tests-acceptance` is run? 
If not, What about splitting the jobs and run standards in the first stage and tests in the second, as you did on the internal GitLab server?
That way it could run parallel and faster, or am I mistaken?